csp_billing_adapter_local/plugin.py

```diff
@@ -1,62 +1,114 @@
+#
+# Copyright 2023 SUSE LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""
+plugin.py is part of csp-billing-adapter-local and provides the local storage
+plugin
+"""
+
 import json
-import os
+import logging
+
+from json.decoder import JSONDecodeError
+from pathlib import Path
 
 import csp_billing_adapter
 
 from csp_billing_adapter.config import Config
 
-cache_file = os.path.expanduser('~/csp-adapter/cache.json')
-csp_config_file = os.path.expanduser('~/csp-adapter/csp-config.json')
+log = logging.getLogger('CSPBillingAdapter')
+
+LOCAL_DIR = '.local'
+PRODUCT_DIR = 'csp-adapter'
```

Comment:
let's move this to csp-billing-adapter to match name.

Reply:
```suggestion
PRODUCT_DIR = 'csp-billing-adapter'
```